ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -51,7 +62,7 @@ interface InstitutionEditState {
 }
 
 let title = 'Add new Institution';
-let institutionToEdit;
+let institutionToEdit: Institution;
```

Comment:
Existing issue, but... what? Declaring these variables statically makes no sense.

Could you please add a TODO here to fix these two variables? I created a ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-7127